test/test-pages.js

```diff
@@ -0,0 +1,194 @@
+    });
+    it("**", function () {
+      expect(fhir.fhirconvert("a**b", vars)).to.equal("%a.power(%b)");
+    });
+  });
+
+  context("Functions", function () {
+    it("CEILING", function () {
+      expect(fhir.fhirconvert("CEILING(a)", vars)).to.equal("(%a).ceiling()");
+    });
+    it("CEILING Exp", function () {
+      expect(fhir.fhirconvert("CEILING(a+b*3)", vars)).to.equal(
+        "(%a+%b*3).ceiling()"
+      );
+    });
+    it("FLOOR", function () {
+      expect(fhir.fhirconvert("FLOOR(a)", vars)).to.equal("(%a).floor()");
+    });
+    it("FLOOR Exp", function () {
+      expect(fhir.fhirconvert("FLOOR(a+b*3)", vars)).to.equal(
+        "(%a+%b*3).floor()"
+      );
+    });
+    it("ABS", function () {
+      expect(fhir.fhirconvert("ABS(a)", vars)).to.equal("(%a).abs()");
+    });
+    it("ABS Exp", function () {
+      expect(fhir.fhirconvert("ABS(a+b*3)", vars)).to.equal(
+        "(%a+%b*3).abs()"
+      );
+    });
+    it("SQRT", function () {
+      expect(fhir.fhirconvert("SQRT(a)", vars)).to.equal("(%a).sqrt()");
+    });
+    it("SQRT Exp", function () {
+      expect(fhir.fhirconvert("SQRT(a+b*3)", vars)).to.equal(
+        "(%a+%b*3).sqrt()"
+      );
+    });
+    it("TRUNCATE", function () {
+      expect(fhir.fhirconvert("TRUNCATE(a)", vars)).to.equal(
+        "(%a).truncate()"
+      );
+    });
+    it("TRUNCATE Exp", function () {
+      expect(fhir.fhirconvert("TRUNCATE(a+b*3)", vars)).to.equal(
+        "(%a+%b*3).truncate()"
+      );
+    });
+    it("EXP", function () {
+      expect(fhir.fhirconvert("EXP(a)", vars)).to.equal("(%a).exp()");
+    });
+    it("EXP Exp", function () {
+      expect(fhir.fhirconvert("EXP(a+b*3)", vars)).to.equal(
+        "(%a+%b*3).exp()"
+      );
+    });
+    it("LN", function () {
+      expect(fhir.fhirconvert("LN(a)", vars)).to.equal("(%a).ln()");
+    });
+    it("LN Exp", function () {
+      expect(fhir.fhirconvert("LN(a+b*3)", vars)).to.equal("(%a+%b*3).ln()");
+    });
+    it("LOG", function () {
+      expect(fhir.fhirconvert("LOG(a, b)", vars)).to.equal("(%b).log(%a)");
+    });
+    it("LOG Exp", function () {
+      expect(fhir.fhirconvert("LOG(a+c, a+b*3)", vars)).to.equal(
+        "(%a+%b*3).log(%a+%c)"
+      );
+    });
+  });
+
+  context("Nested Functions", function () {
+    it("Nested 1", function () {
+      expect(fhir.fhirconvert("CEILING(FLOOR(ABS(a+b*3)))", vars)).to.equal(
+        "(((%a+%b*3).abs()).floor()).ceiling()"
+      );
+    });
+    it("Nested 2", function () {
+      expect(fhir.fhirconvert("CEILING(LOG(2, ABS(a+b*3)))", vars)).to.equal(
+        "(((%a+%b*3).abs()).log(2)).ceiling()"
+      );
+    });
+  });
+
+  context("Validation", function () {
+    it("Non var", function () {
+      expect(fhir.fhirconvert("a+b+z", vars)).to.equal(null);
+    });
+    it("Non func", function () {
+      expect(fhir.fhirconvert("NOTAFUNCTION(a+b)", vars)).to.equal(null);
+    });
+    /*
```

Comment:
Please remove the commented out code for this PR.

Reply:
fixed